src/tests/Myr.test.js

```diff
@@ -752,13 +752,6 @@ describe("Component Animations", () => {
 });
 
 describe("Other Myr functionality", () => {
-    it("should add a model", () => {
-        myr.reset();
-        myr.assets = [];
-        myr.addCModel();
-        expect(myr.assets).toContainEqual({ id: "c-obj", src: "/img/c.obj" });
-    });
-
```

Comment:
Is there a reason why this test is being dropped?

Reply:
This is testing addCModel function from MYR API which create and it's deleted in MYR.js as well since it's not documented anywhere. There aremore APIs that are not documented anywhere but I wasn't sure whether if I should removed it so I moved it down in MYR.js